hoomd/hpmc/ComputeSDF.h

```diff
@@ -149,14 +149,17 @@ template<class Shape> class ComputeSDF : public Compute
 
     //! Return an sdf
     virtual pybind11::array_t<double> getSDF();
+    virtual pybind11::array_t<double> getSDF_expansion();
```

Comment:
Follow the naming conventions outline in https://hoomd-blue.readthedocs.io/en/v3.4.0/style.html